flow/queue/exceptions.py

```diff
@@ -0,0 +1,42 @@
+class InvalidAid(Exception):
```

Comment:
Why bother with exception handling if you do nothing with the exceptions? Is it just for debugging? What happens if this program is run and one of these exceptions comes up?